test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGeneratorVerifyTests/SwaggerGeneratorVerifyTests.cs

```diff
@@ -224,14 +224,15 @@ public Task ActionWithRequiredMember()
 #endif
 
     private static SwaggerGenerator Subject(
-        IEnumerable<ApiDescription> apiDescriptions,
-        SwaggerGeneratorOptions options = null,
-        IEnumerable<AuthenticationScheme> authenticationSchemes = null)
+            IEnumerable<ApiDescription> apiDescriptions,
+            SwaggerGeneratorOptions options = null,
+            IEnumerable<AuthenticationScheme> authenticationSchemes = null,
+            List<ISchemaFilter> schemaFilters = null)
     {
         return new SwaggerGenerator(
             options ?? DefaultOptions,
             new FakeApiDescriptionGroupCollectionProvider(apiDescriptions),
-            new SchemaGenerator(new SchemaGeneratorOptions(), new JsonSerializerDataContractResolver(new JsonSerializerOptions())),
+            new SchemaGenerator(new SchemaGeneratorOptions() { SchemaFilters = schemaFilters ?? [] }, new JsonSerializerDataContractResolver(new JsonSerializerOptions())),
```

Comment:
@martincostello  I have not removed this since it's a good addition here, if we change the test to Verify ones then this does not need to be done

Reply:
Finally I have managed to get the build working